lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -164,13 +164,18 @@ sub refresh_size {
     return $new_size;
 }
 
-sub hidden {
+sub is_type_hidden {
```

Comment:
Seems weird to have this as a public function in a result class.

Reply:
If `OpenQA::Setup` wasn't so chaotic i would say just put it there as a config utility function.

Reply:
Yes, it would be good finding a better place for this function. But for now this is likely the least problem with that draft.